homeassistant/components/apple_tv/__init__.py

```diff
@@ -180,8 +180,11 @@ def connection_lost(self, _):
 
         This is a callback function from pyatv.interface.DeviceListener.
         """
-        _LOGGER.warning('Connection lost to Apple TV "%s"', self.atv.name)
+        _LOGGER.warning(
+            'Connection lost to Apple TV "%s"', self.config_entry.data.get(CONF_NAME)
```

Comment:
`connection_lost` and `connection_closed` are very similar, is there an opportunity to make these more DRY?

Reply:
Yes, I'll bring my towel and dry it up!

Reply:
Done.